.github/workflows/deployDev.yml

```diff
@@ -0,0 +1,38 @@
+name: Deploy dev build to Amazon S3 Dev
```

Comment:
Let's use small caps by convention and no need to add "S3 Dev" at the end. Just:

```
deploy dev build to aws
```